test/model/src/driver/kv_workload_generator.cpp

```diff
@@ -60,7 +60,13 @@ kv_workload_generator_spec::kv_workload_generator_spec()
     truncate = 0.005;
 
     checkpoint = 0.02;
-    checkpoint_crash = 0.002;
+    /*
+     * FIXME-WT-14395 - The last crash point in the checkpoint code (used by test/model's
+     * checkpoint_crash) triggers after the checkpoint txn commits but before the turtle file is
+     * updated. This causes inconsistent behaviour when logging is enabled. For now checkpoint_crash
+     * has been disabled. Once resolved, return checkpoint_crash probability value to 0.002.
+     */
+    checkpoint_crash = 0;
```

Comment:
@mariammojid, @raviprakashgiri29 , is it possible to keep `0.002` but not enable the crash point that is causing issues so we keep some coverage?